functional-tests/src/test/resources/GebConfig.groovy

```diff
@@ -0,0 +1,127 @@
+/*
+  This is the Geb configuration file.
+  See: http://www.gebish.org/manual/current/#configuration
+*/
+
+import org.openqa.selenium.Dimension
+import org.openqa.selenium.chrome.ChromeDriver
+import org.openqa.selenium.chrome.ChromeOptions
+import org.openqa.selenium.firefox.FirefoxDriver
+import org.openqa.selenium.firefox.FirefoxOptions
+import org.openqa.selenium.ie.InternetExplorerDriver
+import org.openqa.selenium.edge.EdgeDriver
+import org.openqa.selenium.safari.SafariDriver
+import org.openqa.selenium.remote.DesiredCapabilities
+import org.openqa.selenium.remote.RemoteWebDriver
+import listeners.SessionIdHolder
+
+
+// Allows for setting you baseurl in an environment variable.
+// This is particularly handy for development and the pipeline
+Map env = System.getenv()
+baseUrl = env['BASE_URL']
+if (!baseUrl) {
+  baseUrl = "https://eagle-mem-mmti.pathfinder.gov.bc.ca/"
+}
+
+USERNAME = env['BROWSERSTACK_USERNAME']
+AUTOMATE_KEY = env['BROWSERSTACK_TOKEN']
+DEBUG = env['DEBUG_MODE']
+
+if (!USERNAME || !AUTOMATE_KEY)
+    throw RuntimeError('BROWSERSTACK_USERNAME and BROWSERSTACK_TOKEN are required');
+
+waiting {
+  timeout = 20
+  retryInterval = 0.5
+}
+
+atCheckWaiting = [20, 0.5]
+
+String buildId = SessionIdHolder.instance.buildId
+
+environments {
+
+  // local dev use
+  chrome {
+    driver = {
+      ChromeOptions o = new ChromeOptions()
+      o.addArguments("window-size=1600,900")
+      new ChromeDriver(o);
+    }
+  }
+
```

Comment:
Maybe we can add Internet Explorer + Safari eventually?

Reply:
Yeah it should be pretty straight forward to add configs for them. And I think Browserstack has configuration that lets us easily tweak IE settings that affect the tests (Security settings not allowing selenium drivers and such).

Reply:
maybe edge higher priority than ie?